### DISCUSSION TITLE
Error post update

### DISCUSSION THREAD

**Discussion by WPTechnician** [Security-Onion-Solutions/securityonion#4701] [Unsupported Versions]
Hi:
I just updated from 2.3.50 to 2.3.60 and when the highstate runs it throws this error:

Running a highstate. This could take several minutes.
local:
    Data failed to compile:
----------
    Rendering SLS 'base:ssl' failed: Jinja variable list object has no element 0
Soup failed with error 1: Unhandled error

I tried manually running the highstate command and receive the same error.  Has anyone else seen this?  Thanks!

**Comment by joeb1kenobe** [comment]
I received the same error. I waited about 2 hours and verified with docker
ps that I was running 2.3.60 images. After that, Highstate ran fine.

On Mon, Jul 5, 2021 at 12:09 PM WPTechnician ***@***.***>
wrote:

> Hi:
> I just updated from 2.3.50 to 2.3.60 and when the highstate runs it throws
> this error:
> Running a highstate. This could take several minutes.
> local:
> Data failed to compile:
>
> Rendering SLS 'base:ssl' failed: Jinja variable list object has no element 0
>
> Soup failed with error 1: Unhandled error
>
> I tried manually running the highstate command and receive the same error.
> Has anyone else seen this? Thanks!
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/4701>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAD6W6RA26YXFZYFVM66P5TTWHKNJANCNFSM472774FQ>
> .
>
-- 
Joe Brown
